packages/react-components/src/organisms/UserProfileBackground.tsx

```diff
@@ -1,14 +1,16 @@
 import { useContext } from 'react';
 import { css } from '@emotion/react';
-import { UserResponse, UserTeam } from '@asap-hub/model';
+import { UserResponse, UserTeam, Lab } from '@asap-hub/model';
 import { network, sharedResearch } from '@asap-hub/routing';
 import { UserProfileContext } from '@asap-hub/react-context';
 
 import { Card, Headline2, Headline3, Link } from '../atoms';
 import { mobileScreen, perRem } from '../pixels';
 import UserProfilePlaceholderCard from './UserProfilePlaceholderCard';
+import { getListStrWithSuffix, capitalizeText } from '../utils';
 
-type UserProfileBackgroundProps = UserTeam & Pick<UserResponse, 'firstName'>;
+type UserProfileBackgroundProps = UserTeam &
+  Pick<UserResponse, 'firstName'> & { labs?: Lab[] };
```

Comment:
we can pick the `labs` from the `UserResponse` type

Reply:
My thought was to set `labs` as optional because there are components that use `UserProfileBackground` but don't render labs, like `UserProfileStaffBackground`